packages/fxa-auth-server/lib/senders/emails/partials/cadReminder/index.ts

```diff
@@ -8,13 +8,13 @@ export const cadReminder = `
   <mj-include path="./lib/senders/emails/css/cadReminder/index.css" type="css" css-inline="inline" />
   <mj-section>
     <mj-column>
-      <mj-text css-class="header-text"><span data-l10n-id="sync-reminder"><%- headerText %></span></mj-text>
+      <mj-text css-class="header-text"><span data-l10n-id="<%- templateName %>-title"><%- headerText %></span></mj-text>
       <mj-image css-class="sync-logo" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
     </mj-column>
   </mj-section>
   <mj-section>
     <mj-column>
-      <mj-text css-class="primary-text"><span data-l10n-id="sync-description"><%- bodyText %></span></mj-text>
+      <mj-text css-class="primary-text"><span data-l10n-id="<%- templateName %>-description"><%- bodyText %></span></mj-text>
     </mj-column>
   </mj-section>
```

Comment:
One super minor thing I noticed that we maybe didn't catch the first time, this is the old version of `cadReminder`:

<img width="408" alt="image" src="https://user-images.githubusercontent.com/13018240/129124165-8e174177-787a-4b6b-bd9a-8e29120f5b82.png">


This is the new version:
<img width="459" alt="image" src="https://user-images.githubusercontent.com/13018240/129124203-b170fe38-6b21-4580-8c00-3f756a4693f1.png">

I think we need more space between the the buttons and the "or, install this on..." text.